app/setup.py

```diff
@@ -178,6 +183,19 @@ def apply_caching(response):  # pylint: disable=unused-variable
 
         return response
 
+    @application.after_request
+    def response_minify(response):  # pylint: disable=unused-variable
```

Comment:
Worth adding a test to ensure this is working.

Reply:
What would the test include? checking that there are no spaces? is there much value there?

Reply:
We could check the `content-length` in the header request is smaller when the minify is enabled?. And `Content-Encoding: gzip` if that is available?.

Reply:
We would essentially be testing a configuration flag here?

Reply:
Which we already do?.

Reply:
Also the current testing harness wont support binary responses like we would get with gzip enabled. to support this we would need to re-write the testing harness, which seems like quite an overhead

Reply:
Had a quick fiddle and in the testing, the content-length header is available which is smaller when the minify is set to true. But to explicitly check `gzip` maybe not.